src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -491,13 +493,22 @@ private void GenerateFunctions(IEnumerable<Type> directTypes)
         /// <summary>
         /// Initialize function descriptors from metadata.
         /// </summary>
-        private void InitializeFunctionDescriptors(Collection<FunctionMetadata> functionMetadata)
+        private void InitializeFunctionDescriptors(Collection<FunctionMetadata> functionMetadata, string language)
         {
-            _descriptorProviders = new List<FunctionDescriptorProvider>()
+            _descriptorProviders = new List<FunctionDescriptorProvider>();
+            if (string.IsNullOrEmpty(language))
```

Comment:
nit: Not worth spending a lot of time, but if this code can be made more concise, that would be a plus.